File: server/src/main/java/com/cyberbot/bomberman/server/Application.java
package com.cyberbot.bomberman.server;

import com.badlogic.gdx.ApplicationListener;

public class Application implements ApplicationListener {
    @Override
    public void create() {
        ServerService service = new ServerService(8038);
        new Thread(service).start();
    }

    @Override
    public void resize(int width, int height) {

    }

    @Override
    public void render() {

    }

    @Override
    public void pause() {

    }

    @Override
    public void resume() {

    }

    @Override
    public void dispose() {

    }
}

File: server/src/main/java/com/cyberbot/bomberman/server/Application.kt
package com.cyberbot.bomberman.server

import com.badlogic.gdx.ApplicationListener

class Application : ApplicationListener {
    override fun create() {
        val service = ServerService(8038)
        Thread(service).start()
    }

    override fun resize(width: Int, height: Int) {}
    override fun render() {}
    override fun pause() {}
    override fun resume() {}
    override fun dispose() {}
}